spec/index.bs

```diff
@@ -4233,10 +4233,10 @@ dictionary GPUTextureDescriptor
         Formats in this list must be [=texture view format compatible=] with the texture format.
 
         <div algorithm data-timeline=const>
-            Two {{GPUTextureFormat}}s |format| and |viewFormat| are <dfn dfn for="">texture view format compatible</dfn> if:
+            Two {{GPUTextureFormat}}s |format| and |viewFormat| are <dfn dfn for="">texture view format compatible</dfn> on a given |device| if:
 
             - |format| equals |viewFormat|, or
-            - |format| and |viewFormat| differ only in whether they are `srgb` formats (have the `-srgb` suffix).
+            - |format| and |viewFormat| differ only in whether they are `srgb` formats (have the `-srgb` suffix) and |device|.{{device/[[features]]}} [=list/contains=] {{GPUFeatureName/"core-features-and-limits"}}.
```

Comment:
This is a breaking change, I think we should avoid wording the specification like this